integrationTests/ts/__tests__/suites/suite3_keychange.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Happy path for user key change",
+    "description": "Single user vote with new keys",
+    "numUsers": 1,
+    "numVotesPerUser": 1,
```

Comment:
@weijiekoh I'm going to refactor how we publish messages in the next PR but this should allow us to start writing a few tests. 

One of the main problems is that `numVotesPerUser` is a bit misleading, since technically if a user changes keys they can vote for another option with a different weight.